wfl-slack-app/src/wfl-api.js

```diff
@@ -0,0 +1,98 @@
+var request = require('request');
+
+function postFoodOrder(restaurantId, callback) {
+    let options = {
+        url: process.env.DB_URI + '/food-order',
+        form: {
+            RestaurantId: restaurantId
+        }
+    }
+
+    request.post(options, function(err, res, body) {
+        let json = JSON.parse(body);
+        if (json) {
+            callback(json);
+        }
+    });
+}
+
+function getAllRestaurants(callback) {
+    let options = {
+        url: process.env.DB_URI + '/restaurant'
+    }
+    request.get(options, function(err, res, body) {
+        callback(JSON.parse(body));
+    });
+}
+
+function getFoodOrderById(id, callback) {
+    let options = {
+        url: process.env.DB_URI + '/food-order/' + id
+    }
+    request.get(options, function(err, res, body){
+        callback(JSON.parse(body));
+    });
+}
+
+function getRestaurantByName(name, callback) {
+    let options = {
+        url: process.env.DB_URI + '/restaurant' + '?Name=' + name,
+    }
+
+    request.get(options, function(err, res, body) {
+        let json = JSON.parse(body);
+        if(json.entities && json.entities[0]) {
+            callback(json.entities[0]);
+        }
+        else {
+            callback();
+        }
+    });
+}
+
+module.exports = {
+    getRestaurantByName: getRestaurantByName,
+    postFoodOrder: postFoodOrder,
+    getAllRestaurants: getAllRestaurants,
+    getFoodOrderById: getFoodOrderById,
+    postMenuSelection: function(foodOrderId, name, description, callback) {
```

Comment:
why did you choose to put some functions inline in the exports and some outside?

Reply:
I was referencing some exported functions from other exported functions. Was not sure how to do that without defining the referenced functions outside of module.exports.

Reply:
I think that's fine, but I would then just make all functions defined outside of the object definition block and make them consistent.